app/uploaders/simple_forms_api/form_remediation/uploader.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'simple_forms_api/form_remediation/configuration/base'
+
```

Comment:
I see this class and a few others repeated in your 5 part series. I would recommend putting the PRs that dup this class into draft mode until we can get the first PR merged and rebase the other PRs. Otherwise, there is a lot of duplication ATM